.github/workflows/build.yml

```diff
@@ -21,11 +28,29 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-vagrant-
 
+    - name: Install libvirt
+      if: matrix.provider == 'libvirt'
+      run: |
+        brew install qemu libvirt
+        # disable unsupported security features
+        echo 'security_driver = "none"' >> /usr/local/etc/libvirt/qemu.conf
+        echo "dynamic_ownership = 0" >> /usr/local/etc/libvirt/qemu.conf
+        echo "remember_owner = 0" >> /usr/local/etc/libvirt/qemu.conf
+        /usr/local/opt/libvirt/sbin/libvirtd -d
+        virsh version
+        libvirtd --help
+        virtqemud --help
+        vagrant plugin install vagrant-libvirt
+
+
     - name: Show Vagrant version
       run: vagrant --version
 
     - name: Run vagrant up
-      run: vagrant up
+      run: |
+        virsh version
+        libvirtd --help
```

Comment:
Is this here only for debugging purposes? Same above.

Reply:
This is all debugging. I need to Cleanup this entire thing.